extensions/vscode/package-lock.json

```diff
@@ -1,12 +1,12 @@
 {
   "name": "ftl",
-  "version": "0.1.2",
```

Comment:
I've been getting these changes in my files as well. Is it meant to be commited?

Reply:
Not sure. Something did it automatically, so I just merged it. It might be related to the auto-release work that @gak did.

Reply:
That's meant to be 0.0.0. It's so we don't accidentally publish the extension outside CI.

Reply:
It shouldn't have done it automatically, it was in a commit.